test/functional/plugin/lsp_spec.lua

```diff
@@ -1954,6 +1954,12 @@ describe('LSP', function()
     end)
   end)
 
+  describe('lsp.util.trim.trim_empty_lines', function() 
+    it('properly trims empty lines', function() 
+      eq({{1, 2}}, exec_lua[[ return vim.lsp.util.trim_empty_lines({{ 1, 2 },  nil}) ]])
+    end)
+  end)
+
```

Comment:
Can we make this test more closely mirror a real use case? Maybe something like

```lua
return vim.lsp.util.trim_empty_lines({"", "hello", "", "world", ""})
```

That is, use text instead of arrays of numbers.